rfcs/0102-moderation-team.md

```diff
@@ -0,0 +1,92 @@
+---
+feature: moderation team
+start-date: 2021-08-18
+author: tomberek
+co-authors: blaggacao
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/rfcs/pull/98
+---
+
+# Summary
+[summary]: #summary
+
+Establish a team to perform moderation.
+
+# Motivation
+[motivation]: #motivation
+
+There is not currently any official mechanism for moderation action. It's not
+sustainable to have to call on Graham any time there's a spammer or conflict
+that requires moderation, and we'd like to help the community become more
+self-regulating.
+
+(adopted from #98)
+
+# Detailed design
+[design]: #detailed-design
+
+The Moderation Team is a volunteer group that may receive and evaluate
+applications to the team or alter the composition at any time with approval
+from the NixOS Foundation board. The team's composition and announcements
+should be maintained at
+[https://nixos.org/community/teams/moderation.html](https://nixos.org/community/teams/moderation.html).
+The team shall perform moderation activities for the [community
+discussions](https://nixos.org/community/index.html) on behalf of the NixOS
+Foundation. The team should utilize the [NixOS Foundation
+mission](https://nixos.org/community/index.html) and the following statement
+during their duties:
+
+```
+The NixOS Foundation aims to promote participation without regard to gender,
+sexual orientation, disability, ethnicity, age, or similar personal
+characteristics. We want to strive to create and foster community by providing
+an intentionally welcoming and safe environment where all feel valued and cared
+for, and where all are given opportunity to participate meaningfully. The
+Foundation will work with the community in service of this goal.
+```
+
+ref: [twitter](https://twitter.com/grhmc/status/1390775249424338944)
+
```

Comment:
Team also needs a contact point, and possibly a contact point per platform.

Reply:
I presume the team will post contact points similar to how the marketing team does: https://nixos.org/community/teams/marketing.html

Reply:
Well, if this is one of the things implied by forming a team, maybe spelling it out is a good idea.